Sources/TensorFlow/Core/Tensor.swift

```diff
@@ -51,10 +51,15 @@ extension Tensor: AnyTensor {
 
 public extension Tensor {
     /// The number of dimensions of the `Tensor`.
-    @inlinable
+    // @inlinable
```

Comment:
Why remove the `@inlinable`?

Reply:
Just a temporary hack as `LazyTensorHandle` is not public. We need to make it `@usableFromInline`.